src/interfacers/tmp/EmonHubSmilicsInterfacer.py

```diff
@@ -120,13 +116,13 @@ def _process_rx(self, smilics_dict):
             c.timestamp = time.mktime(datetime.datetime.now().timetuple())
 
             return c
-        except:
+        except Exception:
             return None
 
     def set(self, **kwargs):
         """ Override default settings with settings entered in the config file
         """
-        for key, setting in self._settings.iteritems():
+        for key, setting in self._settings.tems():
```

Comment:
This looks like a typo, should be `items()`.

Reply:
Cheers @bwduncan fixed https://github.com/openenergymonitor/emonhub/commit/1e09d5a7b77aa820ba1dea8f2c7031916b277dd3